lms/security.py

```diff
@@ -104,13 +104,16 @@ def get_policy(request: Request):  # noqa: PLR0911
             return LMSGoogleSecurityPolicy()
 
         if path.startswith(("/api/dashboard")):
-            # For certain routes we only use the google policy in case it resulted
-            # non empty identity.
-            # This is useful for routes that can be used by admin pages users on top of
-            # other type of access
-            policy = LMSGoogleSecurityPolicy()
-            if policy.identity(request):
-                return policy
+            # For the dashboard API we prefer, in this order:
```

Comment:
We can't follow this file order to accomplish the same thing because:

the first match of path to policy will pick the policy even if that results in an empty identity. Ignoring an identity available in other policy.

That would be nice to have **per path** but not for the general case,  ie not applying all policies to all paths until we are lucky, that can be very confusing.